roborio-src/constants.py

```diff
@@ -41,3 +42,9 @@
 
 kDriverControllerPort = 0
 kOperatorControllerPort = 1
+
+#Xbox controller constants
+deadband = 0.15
```

Comment:
All 4 of these constants should conform to the naming convention.  Put a lower-case k in front and then CamelCase after it.